docs/dev/kind.yaml

```diff
@@ -1,5 +1,14 @@
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
+kubeadmConfigPatches:
```

Comment:
Could you comment on why you added this to dev run?

Reply:
This is to enable validating/mutating webhooks on the local kind cluster, in case you might want to test using them. Can remove from this PR if it feels out of scope

Reply:
Could you leave that comment in the PR?